lib/engine/game/base.rb

```diff
@@ -2007,6 +2011,10 @@ def init_hexes(companies, corporations)
         end.flatten.compact
       end
 
+      def blocker_companies
```

Comment:
why is this necessary?

Reply:
I did not want to break games that didn't allow corporations to be blockers.

Reply:
I did the same thing for partition_companies months ago.

Reply:
Removing.